ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/Runner.java

```diff
@@ -228,4 +217,25 @@ private String buildCorsRegexFromConfig() {
       return stringJoiner.toString();
     }
   }
+
+  private HttpServer createServerAndWait(
+      final Vertx vertx, final Handler<HttpServerRequest> requestHandler)
+      throws ExecutionException, InterruptedException {
+
+    final HttpServer httpServer = vertx.createHttpServer(serverOptions);
+    final CompletableFuture<Void> serverRunningFuture = new CompletableFuture<>();
+    httpServer
+        .requestHandler(requestHandler)
+        .listen(
+            result -> {
+              if (result.succeeded()) {
+                serverRunningFuture.complete(null);
+              } else {
+                serverRunningFuture.completeExceptionally(result.cause());
```

Comment:
Does the server failure cause get logged somewhere?

Reply:
Added an additional log line, but this will ultimately bubble up as an initialisation exception - so if logging in debug should get a full stacktrace.